### PR TITLE
Checking if arg is null in RemovePII

### LINKED ISSUE #1406: NullReferenceException in LogHelper.RemovePII when arg is null
According to description, WsFederationMessage FromQueryString should allow "empty or null" queryString. But it fails with NullReferenceException when queryString is null 

```c#
 public static WsFederationMessage FromQueryString(string queryString) {
            LogHelper.LogVerbose(FormatInvariant(LogMessages.IDX22900, queryString));
```
due to issue with LogHelper.RemovePII:
```c#
private static string RemovePII(object arg)
        {
            if (arg is Exception ex && ex.GetType().FullName.StartsWith("Microsoft.IdentityModel.", StringComparison.Ordinal))
                return ex.ToString();

            return arg.GetType().ToString();
        }
```

### PR DESCRIPTION
**Pull request by RojaEnnam**
Fixes #1406

### CODE REVIEW

**Comment by keegan-caruso** [comment]
Looking at the issue description, it might be worth adding a test to WsFederationMessage:FromQueryString with an input of empty string and null to prove that the code matches the description on the method. Specifically the remarks section on FromQueryString.

File: src/Microsoft.IdentityModel.Logging/LogHelper.cs [hunk 1]
```diff
@@ -339,7 +339,7 @@ private static string RemovePII(object arg)
             if (arg is Exception ex && ex.GetType().FullName.StartsWith("Microsoft.IdentityModel.", StringComparison.Ordinal))
                 return ex.ToString();
 
-            return arg.GetType().ToString();
+            return arg?.GetType().ToString();
```
Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Logging/LogHelper.cs]: Can you also add a check in FormatInvariant above.
if args is null, then we will throw a null ref.

Comment (brentschmaltz) [reply src/Microsoft.IdentityModel.Logging/LogHelper.cs]: @RojaEnnam ill make this change and push the commit.

**Review by GeoK — APPROVED** [review]
